Gemfile.lock

```diff
@@ -328,5 +328,8 @@ DEPENDENCIES
   wicked_pdf
   wkhtmltopdf-binary
 
+RUBY VERSION
+   ruby 2.3.0p0
+
 BUNDLED WITH
-   1.11.2
+   1.12.4
```

Comment:
Is this intentional?  Unless it's necessary let's split out upgrades to dependencies into a separate PR.